contactme.css

```diff
@@ -89,43 +85,41 @@
   border: #fff 1px solid;
 }
 
-.imgcontainer-contact{
- margin-top: 3.5rem;
- display: flex;
- justify-content: center;
+.imgcontainer-contact {
+  margin-top: 3.5rem;
+  display: flex;
+  justify-content: center;
 }
 
 @media only screen and (min-width: 720px) {
```

Comment:
- [ ] The about section has a few details that are not matched.
Align the left column with the right column from the top.
![Screenshot from 2021-08-19 12-52-41](https://user-images.githubusercontent.com/65501535/130048432-cf06faea-d7e9-4888-9b03-698b0d252c64.png)

Reply:
Thanks, But am not getting what you mean!